src/lmtp/commands.c

```diff
@@ -308,7 +308,7 @@ address_add_detail(struct client *client, const char *username,
 }
 
 static bool client_proxy_rcpt(struct client *client, const char *address,
-			      const char *username, const char *detail,
+			      const char *username, const char *detail, char delim,
```

Comment:
This originally was const, but I changed it after the earlier comment by @sirainen. I'm willing to change it back if that is the consensus, or if I misinterpreted his earlier comment.